Dockerfile

```diff
@@ -2,8 +2,15 @@ FROM alpine
 
 RUN apk add --no-cache bash tzdata
 
+RUN mkdir /config && \
+    mkdir /certs && \
+    mkdir /db && \
+    mkdir /decisions
+VOLUME ["/config", "/certs", "/db", "/decisions"]
+
 WORKDIR /app
 
 COPY dist/topaz*_linux_amd64_v1/topaz* /app/
 
-ENTRYPOINT ["./topazd"]
+ENTRYPOINT ["./topazd", "run", "-c"]
+CMD ["/config/config.yaml"]
```

Comment:
I would have expected that the entrypoint is only referencing the binary and cmd all the args, as this is not a reusable entry point as-is, you want to be able to project different cmd definitions if needed.

I would propose to change this into:

```
ENTRYPOINT ["./topazd"]
CMD ["run -c /config/config.yaml"]
```